src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -130,6 +130,8 @@ static MultiNode * ApplyCartesianProduct(MultiNode *leftNode, MultiNode *rightNo
  * Local functions forward declarations for subquery pushdown. Note that these
  * functions will be removed with upcoming subqery changes.
  */
+static Node * ReplaceExternParamsInOriginalQuery(Node *inputNode,
```

Comment:
> ResolveExternalParams vs InstantiateExternalParamsAsConsts

`ResolveExternalParams()` sounds good to me.

(Also, as one of your previous comments suggested, I think I've misused `originalQuery` name/concept slightly wrong.)